pytorch_lightning/trainer/properties.py

```diff
@@ -309,6 +310,14 @@ def early_stopping_callbacks(self) -> List[EarlyStopping]:
         """
         return [c for c in self.callbacks if isinstance(c, EarlyStopping)]
 
+    @property
+    def prediction_writer_callbacks(self) -> List[BasePredictionWriter]:
```

Comment:
The return type could be an empty list as well.

Reply:
Done !